doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+
+`PAB simulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_ is a simulator for the PAB that is used for testing a plutus application in a simulated environment. 
+
+   COMMENTS from Andrew that still need to be addressed: 
+   
+      We need more details. How is this different from the emulator? 
+
+      I'm not sure I agree this is not a high priority component. It may be important when we roll out the new testing framework. 
+
+   COMMENTS from Radu: 
+   
+      I think we need to think about it. This would be the component that will be needed for smart contract testing using Quviq's infrastructure. I am not sure what we should do with it. I have some assumptions about its utility, but I would first need to take a closer look at how it is used.
+
+8. Plutus script utils
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus script utils                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-script-utils``                           |
++--------------+---------------------------------------------------------------+
+
+`Plutus script utils <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-script-utils>`_ is a utility library for helping users write Plutus scripts that are to be used on-chain. `Plutus script utils` includes a variety of useful functions for on-chain operations in Plutus scripts. 
+
+It provides a number of utilities including: 
+
+   * hashing functions for Datums, Redeemers and Plutus scripts for any Plutus language version. 
+   * functionality for wrapping the untyped Plutus script with a typed interface. 
+   * utility functions for working with the ScriptContext of a Plutus Script. 
+
+9. Plutus ledger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus ledger                                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-ledger``                                 |
++--------------+---------------------------------------------------------------+
+
+`Plutus ledger <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger>`_ is a set of transitional types that simplify the cardano-api types. 
+It is intended to be a comprehensive, easy-to-use set of types that replicate the current era of `cardano-api <https://input-output-hk.github.io/cardano-node/cardano-api/lib/Cardano-Api.html>`_. 
+It currently considers only the last era. 
+Plutus ledger contains data types and functions that complement `cardano-ledger <https://github.com/input-output-hk/cardano-ledger>`_ related to Plutus. 
```

Comment:
I am not sure if we should add a note here. We've been discussing using hydra's version of cardano-api helpers which AFAIK does the same thing. Does this mean that we are deprecating it. Should we add a note?

Reply:
@raduom @koslambrou @asutherlandus 
Unless you have already implemented hydra's version of cardano-api helpers or have firm plans to implement that right away, I would think it would be better to not add a note for the time being. Please tell me if you disagree. We can update this in the near future if needed.